src/components/Data/Breadcrumb.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <nav class="subnav">
+    <ul>
+      <li :class="{'active': $route.params.index}">
+        <a href="#!"
+           v-link="{name: 'DataIndexes'}">
+          <i class="fa fa-globe" aria-hidden="true"></i>
+          Indexes
+        </a>
+      </li>
+
+      <li :class="{'active': $route.params.collection, 'active in': isRouteActive($route, 'DataCreateCollection')}"
+          v-if="$route.params.index">
```

Comment:
I think we should use a `props` given by its parent instead of using `$route`.
We must avoid to use `$route` `$router` in children components